diagnostic.md

```diff
@@ -5,74 +5,91 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+A back end is used to store dataa and in game example, game states and views.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+
```

Comment:
Basically because SPA is the way to go. There are far more robust front end tools available to us, and even javascript and html using json can produce a more optimal front end than rails' front end capabilities. 

Does that make some sense.